internal/storage/storage.go

```diff
@@ -187,6 +225,10 @@ func (sb *s3Backend) NewFileWriter(filePath string) (io.WriteCloser, error) {
 
 // GetFileSize returns the size of a specific object
 func (sb *s3Backend) GetFileSize(filePath string) (int64, error) {
+	if sb == nil {
+		return 0, fmt.Errorf("Invalid s3Backend")
+	}
+
```

Comment:
Is this necessary?An invalid back-end should have been caught at startup and prevented this point from being reached.